rpmlb/cli.py

```diff
@@ -48,6 +51,11 @@
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
     help='Instructions for custom downloader and builder.',
 )
+@click.option(
+    '--dist', metavar='DIST',
```

Comment:
The command line option `--dist`'s value is not regular expression.
Sorry. I changed the specification from last time I talked with you.

See https://github.com/sclorg/rpm-list-builder/issues/48#issuecomment-315830205 for detail.

Because if the value is regular expression `RE1`, it is impossible that a regular expression `RE1` compares with another regular expression `RE2` in recipe like below case.
I could not imagine the situation in advance.

```
if [[ RE1 =~ RE2 ]]; then
    something.
fi
```

Also user must know their executed distribution platform.
So, command line option `--dist` is just string.
So, `DIST` is good.